### DISCUSSION TITLE
New feed: cannot see activity from users I follow

### DISCUSSION THREAD

**Discussion by davidsbatista** [community/community#67570] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

I can't no longer see what users I follow start, release, forked, etc.

**Comment by bensworth** [comment]
Yep, all I get is
```
That's all for now

You can adjust your
to see more content, or visit [Explore.](https://github.com/explore)
```
even though there have been recent of changes in repositories I follow or star.

**Comment by davidsbatista** [comment]
found the old feed here: `https://github.com/dashboard-feed`

**Reply by vindarel** [reply]
I just saw this user script to get this old feed where it belongs: https://github.com/wangrongding/github-old-feed
